scripts/optimize-png.js

```diff
@@ -32,6 +32,7 @@ imagemin([process.argv[2]], {
   ...options,
   destination: process.argv[3] || Path.dirname(process.argv[2]),
 }).then(files => {
+  // eslint-disable-next-line no-console
```

Comment:
Ideally, the rule should not apply to `scripts/`. I'm not sure if it's easy to configure though, so not a blocker to merge